templates/project.html

```diff
@@ -253,10 +268,12 @@ <h3 class="question-group-title">{{group.title}}</h3>
                     {# The layout of the project page is either a grid of components or a vertical list of components. #}
                     {% if layout_mode == "grid" %}col-sm-3 col-lg-2{% else %}col-xs-12{% endif %}
                     question
+                    question-row-container
 
                     {# Add CSS classes for whether this question is finished or if it's the next item for the user to do. #}
                     {% if q.is_finished %}task_finished{% endif %}
                     {% if q.first_start %}first_start{% endif %}"
+                    style="background-color: white;"
```

Comment:
This should be on the .question CSS class.

Reply:
Another artifact from an experiment.